input/fsh/EX_Example1.fsh

```diff
@@ -243,7 +243,7 @@ Description: "mCODE Example for TNM Clinical Stage Group"
 * status = #final "final"
 * method = NCIT#C146985 "AJCC Cancer Staging Manual 8th Edition"
 * subject = Reference(mCODEPatientExample1)
-* category = ObsCat#survey "survey"
+* category = ObsCat#exam "exam"
```

Comment:
`#exam` appears to be for physical exams, which I don't think is a necessarily good fit for clinical staging.

> Observations generated by physical exam findings including direct observations made by a clinician and use of simple instruments and the result of simple maneuvers performed directly on the patient's body.

Looking at http://hl7.org/fhir/R4/valueset-observation-category.html, I don't see an obviously better option though.

(This applies to multiple files in this PR.)

Reply:
We had a reviewer comment on this. It is up for a block vote today. He felt exam was a better choice, so I chose to go along.